src/content/blog/growing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Working at Shorebird
+author: eseidel
+description: Shorebird is growing!
+date: 2025-08-27
+cover: '1.0-cover.png'
+slug: 'shorebird-is-growing'
+---
+<!-- cspell:words janky -->
+
+11 years ago, a group of us left the Chrome team believing there had to be
+something better than janky web pages, or complicated Java apps to build for
+mobile, and began
+[the journey that would yield Flutter](https://www.youtube.com/watch?v=xqGAC5QCYuQ).
+I would come to lead to lead and grow the Flutter (and eventually Dart) teams at
+Google until leaving in 2022 to start Shorebird. I left Google in part because I
+realized there were problems I wanted to solve for Flutter that couldn't be
+solved within Google and felt that in the long-term Flutter would be best served
+by a company wholly focused on Flutter. I set out to build that company.
+
+Flutter over those 11 years has thrived. It saw early public success with the
+Hamilton App, NuBank, Toyota, Tencent, Alibaba, ByteDance, and Google's wide
+deployments across Assistant, GPay, Classroom, Earth, etc. Flutter now accounts
+for
+[almost a 3rd of all free app submissions on iOS](https://developers.googleblog.com/en/celebrating-flutters-production-era/)
+and is used around the globe by millions of developers and businesses of
+[all shapes and sizes](https://flutter.dev/showcase).
+
+Shorebird, has also found its place in this success. We started with Code Push,
+to address the
+[#1 most up-voted issue](https://github.com/flutter/flutter/issues/14330) on
+Flutter, and are now serving thousands of businesses globally each month,
+helping them update their apps for millions of customers. But there is so much
+more to do, and so much more we can do, to further Flutter and I still believe a
+focused home for Flutter is what lies in our future.
+
+Shorebird's revenues have tripled since the start of this year, and our userbase
+has grown similarly. So much so that our tiny team now struggles to keep up with
+it all. We're even about to launch a
+[second product](https://ci.shorebird.dev/), which surely will stretch our team
```

Comment:
@felangel what's the best URL to use for this now?

Reply:
Probably https://docs.shorebird.dev/ci